schema/system.hcl

```diff
@@ -198,7 +203,16 @@ table "job_history" {
     type    = timestamptz
     default = sql("now()")
   }
+  column "is_pushed" {
+    null    = false
+    default = false
+    type    = bool
+  }
   primary_key {
     columns = [column.id]
   }
+  index "job_history_is_pushed_idx" {
+    columns = [column.is_pushed]
+    where   = "is_pushed IS FALSE"
```

Comment:
This should match the where clause above

Reply:
My bad, fixed it